docs/data_management/transfer/globus.md

```diff
@@ -181,31 +181,52 @@ Error (transfer)
 Endpoint: UAB Research Computing LTS (Long Term Storage aka S3) (184408b4-d04b-4513-9912-8feeb6adcab3)
 Server: m-a201b5.9ad93.a567.data.globus.org:443
 Command: STOR /test.log
-Message: The connection to the server was broken
----
+Message: The connection to the server was broken ---
 Details: an end-of-file was reached\nglobus_xio: An end of file occurred\n
 ```
 
 #### Buckets Must Have Globally Unique Names
 
-When creating new buckets, the name must be unique across all buckets on the system. At first this may sound very restrictive, but it is quite simple to deal with in practice. See our LTS section on [good naming practice](../lts/index.md#avoiding-duplicate-names-for-buckets) for how to avoid duplicate names.
-
-If a duplicate bucket name is entered, a long error message will appear in a small space next to the new bucket name. The message reads like the following, expanded for readability.
+When creating new buckets, the name must be unique across all buckets on the system. If a duplicate bucket name, for example `first-test-bucket`, is entered, a long error message will appear in a small space next to the new bucket name. For readability, the expanded message is shown below.
 
 ![!large error message in small space](images/globus_lts_duplicate_name_error_001.png)
 
+```text
+Remote Endpoint Failure: Path already exists, Error (mkdir)
+Endpoint: UAB Research Computing LTS (Long Term Storage aka S3) (184408b4-d04b-4513-9912-8feeb6adcab3)
+Server: 138.26.220.68:443
+Message: Path '/first-test-bucket/' already exists
+```
+
+At first using unique name across all buckets on the system may sound very restrictive, but it is quite simple to deal with in practice. See our LTS section on [good naming practice](../lts/index.md#avoiding-duplicate-names-for-buckets) for how to avoid duplicate names. For example, if you want to name a bucket `ai-lab` for storing data related to the entire AI lab or a specific dataset, you can append a universally unique identifier (UUID) to the name. To generate a UUID, visit <https://www.uuidgenerator.net/>, and a 16-byte UUID will be automatically generated. You can then copy it and append it to the name `ai-lab`, as shown below.
```

Comment:
Maybe this?

"At first glance, requiring unique names across all buckets on the system may sound very restrictive, but it is necessary for LTS to be as fast as it is. Fortunately, there is an easy way to deal with the limitation."